fili-core/src/main/java/com/yahoo/bard/webservice/metadata/LookupMetadataLoadTask.java

```diff
@@ -157,13 +163,12 @@ protected SuccessCallback buildLookupSuccessCallback() {
 
             pendingLookups.addAll(
                     dimensionDictionary.findAll().stream()
-                            .filter(dimension -> dimension instanceof LookupDimension)
-                            .map(dimension -> (LookupDimension) dimension)
-                            .map(LookupDimension::getNamespaces)
+                            .filter(dimension -> dimension instanceof RegisteredLookupDimension)
```

Comment:
Right. I think it would make more sense to have this loadTask check specifically against *Registered*LookupDimensions.

Reply:
It's not possible that this is working for the lookup dimensions because only the registered lookup dimensions HAVE any metadata at this druid endpoint.  Therefore there's no concern about this breaking existing clients.